logstash-output-email.gemspec

```diff
@@ -25,6 +25,6 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'mail'
 
   s.add_development_dependency 'logstash-devutils'
-  s.add_development_dependency 'rumbster'
+  s.add_runtime_dependency 'rumbster'
```

Comment:
This gem is only used when running the tests and it should be a development dependencies. In what context these tests are failing, locally or jenkins?